rts/Lua/LuaSyncedRead.cpp

```diff
@@ -1686,12 +1686,12 @@ int LuaSyncedRead::GetTeamList(lua_State* L)
 }
 
 
-/***
+/*** Get player IDs
  *
  * @function Spring.GetPlayerList
- * @param teamID integer? (Default: `-1`) to filter by when >= 0
- * @param active boolean? (Default: `false`) whether to filter only active teams
- * @return number[]? list of playerIDs
+ * @param teamID integer? (Default: `-1`) when >= 0 filters by teamID and filters non-spectator players if `active` is true
+ * @param active boolean? (Default: `false`) whether to filter by active players
+ * @return number[]? list of playerIDs. Returns: nil when `teamID` is invalid, empty when player is spectating a replay in unsynced environment
```

Comment:
"empty when player is spectating a replay in unsynced environment" sounds like a bug.

Reply:
Would you be able to help me understand [this line](https://github.com/beyond-all-reason/RecoilEngine/pull/2546/files#diff-6f668d87ca38359e7f207ff7ebfd2af01abc12ce57fe906e63b074426d9d7133R1721)?